runtime/compiler/runtime/StatisticsThread.cpp

```diff
@@ -83,14 +83,24 @@ static int32_t J9THREAD_PROC statisticsThreadProc(void * entryarg)
 
          if ((statisticsThread->getStatisticsFrequency() != 0) && ((crtTime - lastStatistics) > statisticsThread->getStatisticsFrequency()))
             {
+            int32_t cpuUsage = 0, avgCpuUsage = 0, vmCpuUsage = 0;
+            bool isFunctional = false;
+            CpuUtilization *cpuUtil = compInfo->getCpuUtil();
+            if (cpuUtil->isFunctional())
+               {
+               cpuUtil->updateCpuUtil(jitConfig);
+               cpuUsage = cpuUtil->getCpuUsage();
+               avgCpuUsage = cpuUtil->getAvgCpuUsage();
+               vmCpuUsage = cpuUtil->getVmCpuUsage();
+               isFunctional = true;
+               }
             TR_VerboseLog::vlogAcquire();
             TR_VerboseLog::writeLine(TR_Vlog_JITaaS, "Number of clients : %u", compInfo->getClientSessionHT()->size());
             TR_VerboseLog::writeLine(TR_Vlog_JITaaS, "Total compilation threads : %d", compInfo->getNumUsableCompilationThreads());
             TR_VerboseLog::writeLine(TR_Vlog_JITaaS, "Active compilation threads : %d",compInfo->getNumCompThreadsActive());
-            CpuUtilization *cpuUtil = compInfo->getCpuUtil();
-            if (cpuUtil->isFunctional())
+            if (isFunctional)
```

Comment:
Nit-pick: it's not clear what `isFunctional` refers to without reading a bunch of the preceding code. One possibility is to change the name of the variable to `isCpuUtilizationFunctional`, but a better approach is to eliminate this variable and directly test cpuUtil->isFunctional(). This is a getter that should be easily inlined by the c++ compiler, so there is no runtime overhead.